src/callbacks_step/save_restart.jl

```diff
@@ -141,6 +141,17 @@ function load_timestep(restart_file::AbstractString)
     end
 end
 
+"""
+    load_timestep!(integrator, restart_file::AbstractString)
+
+Load the time step number (`iter` in OrdinaryDiffEq.jl) saved in a `restart_file`. And set
+the number of accepted steps (`stats.naccept` in OrdinaryDiffEq.jl) equal to the time step number.
```

Comment:
```suggestion
Load the time step number saved in a `restart_file` and assign it to both the time step
number and and the number of accepted steps
(`iter` and `stats.naccept` in OrdinaryDiffEq.jl, respectively) in `integrator`.
```
I feel like this is slightly clearer, what do you think?

Reply:
Thank you for a review! It's really better, it was difficult for me to formulate!